reference/5.0/Microsoft.PowerShell.Archive/Compress-Archive.md

```diff
@@ -225,9 +251,25 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Confirm
```

Comment:
Confirm and WhatIf are both CommonParameters.  Is there a need for them to be additionally defined here?

Reply:
They are not the official list of common parameters. You have to code for the behavior. You don't for the other common parameters.

Reply:
Ahhh. It's confusing as they are listed and described in about_CommonParameters.